daffodil-test/src/test/scala/org/apache/daffodil/section00/general/TestSchemaWithoutDFDLNamespace.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.daffodil.section00.general
+
+import org.apache.daffodil.tdml.Runner
+import org.junit.{AfterClass, Test}
+
+object TestSchemaWithoutDFDLNamespace {
+  val testDir = "/org/apache/daffodil/section00/general/"
+  val runner = Runner(testDir, "testSchemaWithoutDFDLNamespace.tdml")
+
+  @AfterClass def shutDown(): Unit = {
+    runner.reset
```

Comment:
For future reference, we don't want new code to have easily fixable IDE warnings (val runner: Runner = ..., and runner.reset()).  IDEA even tries to prevent you from committing code with such warnings so other devs changing the same file will have to fix the warnings or answer yes, commit anyway.  Scala 3 migration will need these fixes anyway if my memory is correct.

Reply:
I'm not seeing any warnings over here. Can you let me know what the warnings are and I'll see if I can get those turned on. Thanks.

Reply:
I'm not sure why you wouldn't see these warnings since IDEA should be enabling some kinds of warnings out of the box already.  You can check your IDEA settings: Open File -> Settings, type scala into the search box, hit the Enter key, and click on Editor -> Inspections to see all the enabled Scala warnings (Build -> Compiler -> Scala Compiler also has an Enable warnings setting which should be enabled too).  

Another possibility is that you're not seeing the warnings because IDEA is very subtle in showing them (a yellow bar in the right gutter, a count of warnings in the upper right corner, and you have to click on the count to see a list of warnings in the Problems tab).

Reply:
I'm seeing them now. They were there but very subtle as you indicated. I'm brand new to Scala and IDEA so it take me a bit to learn these. Thanks for the info. 

I've updated the tests to include the recommended changes from the warnings.